### ISSUE TITLE
[Enhancement] Improve robustness against buff popups

### ISSUE THREAD

**Issue by Cyrexxis** [Cyrexxis/RokTracker#8]
**Is your feature request related to a problem? Please describe.**
Currently, it can happen that the scan crashes if an alliance or kingdom buff gets launched during the scan, because you get a popup that blocks the stats.

**Describe the solution you'd like**
A solution to that would be to always check if the screen looks like it should - before trying to get the stats - and if not / if a popup is detected, click it away.

**Comment by Cyrexxis** [comment]
currently it seems that the problem i tried to fix here is more a problem with slow network than it is a problem with buffs.

**Comment by leonyonz** [comment]
yeah, sometimes it stuck when buff appear on the screen. the solution i think you can add notification / alert who sent by email or telegram bot so user can re-run the bot manually.

**Comment by Cyrexxis** [comment]
That would be an idea. However, the difficult thing is to detect the buff popup. Once that works, it would be no problem at all to let the bot click the popup away and continue on its own.
